plugins/META-INF/MANIFEST.MF

```diff
@@ -18,42 +18,71 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.datatools.connectivity.ui;resolution:=optional,
  org.eclipse.datatools.connectivity.ui.dse;resolution:=optional,
  org.eclipse.equinox.security,
- org.eclipse.osgi
+ org.eclipse.datatools.enablement.oracle;bundle-version="1.2.100",
+ org.eclipse.datatools.enablement.oracle.dbdefinition;bundle-version="1.2.100",
+ org.eclipse.datatools.enablement.oracle.ui;bundle-version="1.2.100"
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
-Bundle-ClassPath: lib/sdk/oci-java-sdk-full-1.23.0.jar,
- lib/thirdparty/commons-lang3-3.8.1.jar,
- lib/thirdparty/commons-io-2.6.jar,
- lib/thirdparty/commons-codec-1.13.jar,
- lib/thirdparty/guava-27.1-jre.jar,
- lib/thirdparty/json-smart-2.3.jar,
- lib/thirdparty/jersey-client-2.27.jar,
- lib/thirdparty/jersey-common-2.27.jar,
- lib/thirdparty/jersey-entity-filtering-2.27.jar,
- lib/thirdparty/jersey-media-json-jackson-2.27.jar,
- lib/thirdparty/jersey-hk2-2.27.jar,
- lib/thirdparty/slf4j-api-1.7.26.jar,
- lib/thirdparty/aopalliance-repackaged-2.5.0-b42.jar,
- lib/thirdparty/bcpkix-jdk15on-1.60.jar,
- lib/thirdparty/bcprov-jdk15on-1.60.jar,
- lib/thirdparty/hk2-api-2.5.0-b42.jar,
- lib/thirdparty/hk2-locator-2.5.0-b42.jar,
- lib/thirdparty/hk2-utils-2.5.0-b42.jar,
- lib/thirdparty/jackson-annotations-2.10.1.jar,
- lib/thirdparty/jackson-core-2.10.1.jar,
- lib/thirdparty/jackson-databind-2.10.1.jar,
- lib/thirdparty/jackson-datatype-jdk8-2.10.1.jar,
- lib/thirdparty/jackson-datatype-jsr310-2.10.1.jar,
- lib/thirdparty/jackson-module-jaxb-annotations-2.8.10.jar,
- lib/thirdparty/javassist-3.24.0-GA.jar,
- lib/thirdparty/javax.annotation-api-1.2.jar,
- lib/thirdparty/javax.inject-2.5.0-b42.jar,
- lib/thirdparty/javax.ws.rs-api-2.1.jar,
- lib/thirdparty/nimbus-jose-jwt-8.2.1.jar,
- lib/thirdparty/validation-api-1.1.0.Final.jar,
- lib/thirdparty/resilience4j-circuitbreaker-1.2.0.jar,
- .
+Bundle-ClassPath: .
+  ,lib/sdk/oci-java-sdk-circuitbreaker-2.8.0.jar
```

Comment:
I think it's more elegant to put the comma at the end of the line like it used to be

Reply:
I had to take this approach because ant doesn't have a built-in way that I could find to fix up a string.  I would end up with a dangling comma from list concatenation at the end (an error from Eclipse's point of view) and no way to chop it off without introducing additional ant dependencies either for javascript or some other external language just to chop one comma.